### PR TITLE
Fixes [8825]

### PR DESCRIPTION
**Pull request by Probe1**
Adds 

```

use_power = 1   
idle_power_usage = 5
active_power_usage = 100

```

to the cereal maker, grill, oven, still, and deep fryer.

Fixes https://github.com/d3athrow/vgstation13/issues/8825

### CODE REVIEW

**Comment by clusterfack** [comment]
Does it though?

**Comment by Probe1** [comment]
You got me, I hit pull while waiting for dream daemon to work.  What did I miss here?

**Comment by Intigracy** [comment]
look here:
https://github.com/Probe1/vgstation13/blob/Kitchen_Power/code/modules/food/cooking_machines.dm#L54
These vars are already set.

**Comment by Probe1** [comment]
```

if(stat & (NOPOWER | BROKEN))

```

File: data/investigate/singulo.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<small>21:49 [0x201b9e3] (393,125,4)</small> || Energy Cannon regained power and turned <font color='green'>on</font><br />
```
Comment (ghost) [inline data/investigate/singulo.html:1]: Remove this.

**Comment by Probe1** [comment]
cocks

**Comment by ghost** [comment]
Make sure the spit doesn't need power, it inherits from the grill.

**Comment by Probe1** [comment]
It has use_power = 0, I checked.

**Comment by gbasood** [comment]
> closed

**Comment by Probe1** [comment]
I was tired and committed when trying to override my previous commit with Bleeding Edge + the actual fix.  I made a new PR out of shame.

**Pull request closed without merging**